script/jenkins/windows-test.ps1

```diff
@@ -57,5 +65,6 @@ echo "System testing $env:beat"
 $packages = $(go list ./... | select-string -Pattern "/vendor/" -NotMatch | select-string -Pattern "/scripts/cmd/" -NotMatch)
 $packages = ($packages|group|Select -ExpandProperty Name) -join ","
 exec { go test -race -c -cover -covermode=atomic -coverpkg $packages } "go test FAILURE"
-Set-Location -Path tests/system
-exec { nosetests --with-timer --with-xunit --xunit-file=../../build/TEST-system.xml } "System test FAILURE"
+
+echo "Running python tests"
+exec { mage pythonUnitTest } "System test FAILURE"
```

Comment:
See the current behaviour for the master branch:
- https://github.com/elastic/apm-server/blob/2800966ffdc39b201cf1b933dfe286764be1a6ef/script/jenkins/windows-test.ps1#L65-L69

Caused by https://github.com/elastic/apm-server/commit/fe9981060a729073e26bfb937c53558dc45504b7#diff-a64e2e5f49641718219ec8048883e708L60-L61